app/main.py

```diff
@@ -1,8 +1,38 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
+                 clean_mark: int, brand: str) -> int or str:
```

Comment:
Are you sure that this method returns int or str?) 
The same in other similar cases